### ISSUE TITLE
Change name of "context

### ISSUE THREAD

**Issue by stevvooe** [containerd/continuity#78]

**Comment by kzys** [comment]
+1

@AkihiroSuda @dmcgowan Thoughts?

**Comment by dmcgowan** [comment]
Yes, although the root package hasn't been touched in awhile. If you have plans to resurrect some of the work here, it is worth discussing and looking at some of the old designs and work.

**Comment by kzys** [comment]
Sure. I already took a few easy ones (such as #190 and #191) but may check other issues.

Does it have design docs somewhere?
